frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/InferenceServiceServingRuntimeSection.tsx

```diff
@@ -58,16 +58,17 @@ const InferenceServiceServingRuntimeSection: React.FC<
         }))}
         toggleProps={{ id: 'inference-service-model-selection' }}
         isFullWidth
-        value={data.servingRuntimeName}
```

Comment:
I think maybe we should still pass this value in in case SimpleSelect changes in the future to make it matter. The fact that passing `toggleLabel` makes `value` no longer used is not part of the interface contract of the component, just an implementation detail that could change.

Reply:
Thanks, updated